neo/swf/SWF_Sprites.h

```diff
@@ -95,6 +95,7 @@ class idSWFSprite
 	{
 		swfTag_t		tag;
 		idSWFBitStream	stream;
+		swfSpriteCommand_t& operator=(swfSpriteCommand_t&& other) = default;
```

Comment:
Why is this one needed?

Reply:
Ah — I think I got a compile error for a missing move constructor for `swfSpriteCommand_t` before I had created one for `idSWFBitStream`, because the default one couldn't be instantiated. Thanks for the catch, I'll re-submit without this.